File: _Contabilidad/MantenimientoCtasContables/parametrosEmpresa.vb
Imports System.Data
Public Module emp
    Friend InvUltAnu As DateTime = New DateTime(1900, 1, 1)
    Friend RolCodMay As DateTime = New DateTime(1900, 1, 1)
    Friend CtaNumDigNivel As String = "0000000"
    Friend CtaNumNiveles As Int32 = 0
    Friend NumeroDigitos As Int32 = 0
    Friend Acf As Boolean = False
    Friend inv As Boolean = False
    Friend rol As Boolean = False
    Friend autorizacion As String = ""
    Public Sub cargarParametrosEmpresa()
        Dim DatEmp As DataTable = DattCom.datosEmpresa.leeParametrosEmp(" DefCta_NumDigNivel,DefCta_NumNiveles,Par_Numerodigitos,RolCodMay")
        If DatEmp.Rows.Count > 0 Then
            'InvUltAnu = Convert.ToDateTime(DatEmp.Rows(0).Item("InvUltAnu"))
            RolCodMay = Convert.ToDateTime(DatEmp.Rows(0).Item("RolCodMay"))
            CtaNumDigNivel = DatEmp.Rows(0).Item("DefCta_NumDigNivel").ToString()
            CtaNumNiveles = Convert.ToInt32(DatEmp.Rows(0).Item("DefCta_NumNiveles"))
            NumeroDigitos = Convert.ToInt32(DatEmp.Rows(0).Item("Par_Numerodigitos"))
        End If
    End Sub
End Module

File: _Contabilidad/MantenimientoCtasContables/parametrosEmpresa.cs
using System;

namespace MantCtb
{
    public static class emp
    {
        internal static DateTime InvUltAnu = new DateTime(1900, 1, 1);
        internal static DateTime RolCodMay = new DateTime(1900, 1, 1);
        internal static string CtaNumDigNivel = 0;
        internal static int CtaNumNiveles = 0;
        internal static int NumeroDigitos = 0;
        internal static bool Acf = false;
        internal static bool inv = false;
        internal static bool rol = false;
        internal static string autorizacion = "";

        public static void cargar()
        {
            var DatEmp = SysEmpDatt.datosEmpresa.leeParametrosEmp(" InvUltAnu, RolCodMay,CtaNumDigNivel,CtaNumNiveles,NumeroDigitos");
            if (DatEmp.Rows.Count > 0)
            {
                InvUltAnu = Convert.ToDateTime(DatEmp.Rows[0]["InvUltAnu"]);
                RolCodMay = Convert.ToDateTime(DatEmp.Rows[0]["RolCodMay"]);
                CtaNumDigNivel = DatEmp.Rows[0]["CtaNumDigNivel"].ToString();
                CtaNumNiveles = Convert.ToInt32(DatEmp.Rows[0]["CtaNumNiveles"]);
                NumeroDigitos = Convert.ToInt32(DatEmp.Rows[0]["NumeroDigitos"]);
            }
        }
    }
}